src/proxy-animation.js

```diff
@@ -382,16 +381,33 @@ function syncCurrentTime(details) {
     return;
 
   if (details.startTime !== null) {
-    const timelineTime = fromCssNumberish(details,
-                                          details.timeline.currentTime);
-    details.animation.currentTime =
-        (timelineTime - details.startTime) *
-            details.animation.playbackRate;
+    const timelineTime = details.timeline.currentTime;
+    if (timelineTime == null)
+      return;
+
+    const timelineTimeMs = fromCssNumberish(details, timelineTime);
+
+    setNativeCurrentTime(details,
+                         (timelineTimeMs - details.startTime) *
+                             details.animation.playbackRate);
   } else if (details.holdTime !== null) {
-    details.animation.currentTime = details.holdTime;
+    setNativeCurrentTime(details, details.holdTime);
   }
 }
 
+// Sets the time of the underlying animation, nudging the time slightly if at
+// a scroll-timeline boundary to remain in the active phase.
+function setNativeCurrentTime(details, time) {
+  const timeline = details.timeline;
+  const atScrollTimelineBoundary =
+      timeline.currentTime &&
+      timeline.currentTime.value == (this.playbackRate < 0 ? 0 : 100);
```

Comment:
Getting an exception here when loaded as a module _(and thus in strict mode)_. Instead of `this.playbackRate` this should be `details.animation.playbackRate`.

Same with two lines down.

Reply:
Fixed.  Added "use strict" to non-CSS demos.  This should make it easier to catch these errors.